swagger.yml

```diff
@@ -3548,31 +3609,51 @@ paths:
             application/json:
               schema:
                 type: object
-                required:
-                  - Features
-                  - Technologies
```

Comment:
Why do you have removed the required specification?

Reply:
I didn't remove it on purpose. when I changed to #/component/schemas/ I forgot to specify Required. Done !